docs/cli/pyats_run.rst

```diff
@@ -103,12 +103,10 @@ the job file, returing the overall results, and emailing you the report.
       --clean-scope {job,task}
                             Specify whether clean runs before job or per task
       --invoke-clean        Clean is only invoked if this parameter is specified.
-      --clean-image  [ ...]
-                            Image files for each device
-      --clean-platform  [ ...]
-                            Image files for each platform
-      --clean-separator     Separator between device/platform & image file in arguments clean-image and
-                            clean-platform
+      --clean-device-image        list of clean images per device in format device:/path/to/image.bin
```

Comment:
Is it a list? Or is it just multiple strings like:
`--clean-device-image PE1:image1 PE2:image2`

same everywhere else